src/Bicep.Core/Semantics/Namespaces/SystemNamespaceSymbol.cs

```diff
@@ -389,9 +404,113 @@ public class SystemNamespaceSymbol : NamespaceSymbol
                 .WithReturnType(LanguageConstants.String)
                 .WithDescription("Returns a value in the format of a globally unique identifier. **This function can only be used in the default value for a parameter**.")
                 .WithFlags(FunctionFlags.ParamDefaultsOnly)
+                .Build(),
+
+            new FunctionOverloadBuilder("loadTextContent")
+                .WithDescription("Load content of specified file into a string. Content loading occurs during compilation, not in runtime. Maximum allowed content size is 131,072 characters (including line endings).")
```

Comment:
>Load content of specified file into a string. Content loading occurs during compilation, not in runtime. [](http://example.com/codeflow?start=34&length=104)

Can we reword the first two sentences to the following?
"Loads the content of the specified file into a string. Content loading occurs during compilation, not at runtime."

Reply:
The next sentence should also say "The maximum...".

Similar comment for the one below.

Reply:
done.